src/test/globalSetup.ts

```diff
@@ -0,0 +1,11 @@
+// Jest expects a single default function to be exported from this file.
+/* eslint-disable import/no-default-export */
+import type { Config } from 'jest';
+
+export default (globalConfig: Config, projectConfig: Config): void => {
```

Comment:
This is a brilliant solution 👨‍🍳💋 (<-- that's supposed to mean "chefkiss")

Also I don't know if you use StackOverflow but if you do and you want internet points you have found a great answer to: https://stackoverflow.com/questions/73058368/is-it-possible-to-check-if-jest-silent-is-enabled-in-runtime